### ISSUE TITLE
How to build in portable mode?

### ISSUE THREAD

**Issue by stefan2904** [hyperledger/ursa#131]
I am currently playing around with this lib. While doing so I tried to build in portable mode.

The recommended way of doing so, according to the README:

```
cargo build --release --no-default-features --features=portable
```

If compiling that way, cargo fails with:

> error: --features is not allowed in the root of a virtual workspace
> note: while this was previously accepted, it didn't actually do anything

So I checked your [CI file](https://github.com/hyperledger/ursa/blob/master/azure-pipelines.yml#L58), in which you first compile the whole project with default features, and than just _libzmix_ with portable feature enabled:

```
cargo build
cargo build --manifest-path=libzmix/Cargo.toml --no-default-features --features=portable
```

Does that make sense? Is that correct? 
If yes, I propose to add that information to the README.

(I can do a PR, but I don't understand the library well enough to be sure it is the correct way of doing this.)

**Comment by mikelodder7** [comment]
Yes we do need to update the documentation since each subproject can be built for portable.

If you need just ursa primitives, you would build it
`cargo build --manifest-path=libursa/Cargo.toml --release --no-default-features --features=portable`

The other command is for zmix.

**Comment by brentzundel** [comment]
next steps:
1. make sure the posted solution still works
2. update the documentation
